pkg/scheduler/apis/config/types_test.go

```diff
@@ -200,3 +201,95 @@ func TestPluginsApply(t *testing.T) {
 		})
 	}
 }
+
+func TestPluginsContains(t *testing.T) {
```

Comment:
@ahg-g I've added several unit tests here. Do we still need the compatibility test?

Reply:
The compatibility test tests the configuration behavior of predicates and priorities.

Reply:
The compatibility test should verify the behavior of the configuration API, not just the Policy API. In this case, the test would verify that if the configuration didn't specify those filters, they should still be included in the final scheduler configuration.

Reply:
> The compatibility test should verify the behavior of the configuration API, not just the Policy API. In this case, the test would verify that if the configuration didn't specify those filters, they should still be included in the final scheduler configuration.

I see your point. What I mean is, all of the tests are for predicates and priorities, do we need to create tests with plugin configuration in the compatibility tests now?

Reply:
Yes, we should have compatibility tests now for plugin-based configurations.

Reply:
> Yes, we should have compatibility tests now for plugin-based configurations.

Sure, can I open another PR to update the compatibility tests first since we have not reached a consensus on what we should do with mandatory plugins?